sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -651,168 +689,6 @@ private void savePropertiesChangesToFile(Properties properties, File file)
-		String version = promptForMissingValueWithOptions(message,
-				null, "distribution artifact", Lists.newArrayList(optionsMap.keySet()), "Please specify %s",
-				REFERENCEAPPLICATION_2_4);
-
-		String artifact = optionsMap.get(version);
-		if (artifact != null) {
-			return artifact;
-		} else {
-			return version;
-		}
-	}
-
-	private String promptForO3Version(Map<String, String> optionsMap, String customMessage) throws MojoExecutionException {
-		String message = customMessage != null ? customMessage : O3_VERSION_PROMPT;
-		String version = promptForMissingValueWithOptions(message,
-				null, "O3 artifact", Lists.newArrayList(optionsMap.keySet()), "Please specify %s",
-				REFERENCEAPPLICATION_O3);
-
-		String artifact = optionsMap.get(version);
-		if (artifact != null) {
-			return artifact;
-		} else {
-			return version;
-		}
-	}
-
-	private Map<String, String> getDistroVersionsOptionsMap(Set<String> versions, VersionsHelper versionsHelper,
-			String optionTemplate, String artifactTemplate) {
-		Map<String, String> optionsMap = new LinkedHashMap<>();
-
-		List<ArtifactVersion> artifactVersions = new ArrayList<>();
-		for (String version : versions) {
-			artifactVersions.add(new DefaultArtifactVersion(version));
-		}
-		for (String version : versionsHelper.getSuggestedVersions(artifactVersions, MAX_OPTIONS_SIZE)) {
-			optionsMap.put(String.format(optionTemplate, version), String.format(artifactTemplate, version));
-			if (optionsMap.size() == MAX_OPTIONS_SIZE)
-				break;
-		}
-		return optionsMap;
-	}
-
-	/**
-	 * Returns a map of options based on the versions of O3
-	 *
-	 * @param versionsHelper   The VersionsHelper object to retrieve the artifact versions from.
-	 * @param optionTemplate   The template for generating option keys in the map.
-	 * @return A LinkedHashMap containing the generated options map.
-	 */
-	private Map<String, String> getO3VersionsOptionsMap(VersionsHelper versionsHelper, String optionTemplate) {
-		Map<String, String> optionsMap = new LinkedHashMap<>();
-
-		{
-			Artifact artifact = new Artifact(SDKConstants.REFAPP_3X_ARTIFACT_ID, "3.0.0", "org.openmrs", "zip");
-			for (ArtifactVersion version : versionsHelper.getAllVersions(artifact, MAX_OPTIONS_SIZE)) {
-				optionsMap.put(String.format(optionTemplate, version.toString()), artifact.getGroupId() + ":" + artifact.getArtifactId() + ":" + version);
-			}
-		}
-
-		if (optionsMap.size() == MAX_OPTIONS_SIZE) {
-			return optionsMap;
-		}
-
-		{
-			Artifact artifact = new Artifact(SDKConstants.REFAPP_DISTRO, "3.0.0", "org.openmrs.distro", "zip");
-			for (ArtifactVersion version : versionsHelper.getAllVersions(artifact, MAX_OPTIONS_SIZE)) {
-				if (!version.toString().endsWith("-SNAPSHOT") && optionsMap.size() < MAX_OPTIONS_SIZE) {
-					optionsMap.put(String.format(optionTemplate, version),
-							artifact.getGroupId() + ":" + artifact.getArtifactId() + ":" + version);
-				}
```

Comment:
This is something that was easier NOT to support in this change, and it didn't seem like something we need anymore now that 3.0.0 has been released.  This particular distro artifact was only ever a snapshot/beta/alpha of 3.0.0, so there should not be any need for it anymore.  Someone could still choose to install it via the custom distribution option, by specifying the full maven coordinate string, but I don't think we need this sitting in the list of versions in the Wizard, so I removed it.